dependabot/build.gradle

```diff
@@ -0,0 +1,112 @@
+    api "org.apache.groovy:groovy-ant:${project['groovy.version']}"
+    api "org.apache.groovy:groovy-cli-picocli:${project['groovy.version']}"
+    api "org.apache.groovy:groovy-console:${project['groovy.version']}"
+    api "org.apache.groovy:groovy-dateutil:${project['groovy.version']}"
+    api "org.apache.groovy:groovy-json:${project['groovy.version']}"
+    api "org.apache.groovy:groovy-macro:${project['groovy.version']}"
+    api "org.apache.groovy:groovy-nio:${project['groovy.version']}"
+    api "org.apache.groovy:groovy-sql:${project['groovy.version']}"
+    api "org.apache.groovy:groovy-swing:${project['groovy.version']}"
+    api "org.apache.groovy:groovy-templates:${project['groovy.version']}"
+    api "org.apache.groovy:groovy-test:${project['groovy.version']}"
+    api "org.apache.groovy:groovy-test-junit5:${project['groovy.version']}"
+    api "org.apache.groovy:groovy-xml:${project['groovy.version']}"
+    api "org.grails:grails-gsp:${project['gsp.version']}"
+    api "org.grails:grails-web-gsp:${project['gsp.version']}"
+    api "org.grails:grails-web-taglib:${project['gsp.version']}"
+    api "org.grails.plugins:gsp:${project['gsp.version']}"
+    api "com.h2database:h2:${project['h2.version']}"
+    api "org.hamcrest:hamcrest-core:${project['hamcrest-core.version']}"
+    api "com.fasterxml.jackson.core:jackson-databind:${project['jackson-databind.version']}"
+    api "jakarta.annotation:jakarta.annotation-api:${project['jakarta.annotation-api.version']}"
+    api "jakarta.inject:jakarta.inject-api:${project['jakarta.inject-api.version']}"
+    api "jakarta.persistence:jakarta.persistence-api:${project['jakarta.persistence-api.version']}"
+    api "jakarta.servlet:jakarta.servlet-api:${project['jakarta.servlet-api.version']}"
+    api "jakarta.xml.bind:jakarta.xml.bind-api:${project['jakarta.xml.bind-api.version']}"
+    api "org.fusesource.jansi:jansi:${project['jansi.version']}"
+    api "com.github.javaparser:javaparser-core:${project['javaparser-core.version']}"
+    api "org.slf4j:jcl-over-slf4j:${project['jcl-over-slf4j.version']}"
+    api "jline:jline:${project['jline.version']}"
+    api "net.java.dev.jna:jna:${project['jna.version']}"
+    api "org.jsoup:jsoup:${project['jsoup.version']}"
+    api "org.mongodb:bson:${project['mongodb.version']}"
+    api "org.mongodb:bson-record-codec:${project['mongodb.version']}"
+    api "org.mongodb:mongodb-driver-core:${project['mongodb.version']}"
+    api "org.mongodb:mongodb-driver-sync:${project['mongodb.version']}"
+    api "org.objenesis:objenesis:${project['objenesis.version']}"
+    api "org.grails.plugins:cache:${project['plugins-cache.version']}"
+    api "org.grails.plugins:fields:${project['plugins-fields.version']}"
+    api "org.grails.plugins:geb:${project['plugins-geb.version']}"
+    api "org.grails.plugins:gorm.hibernate5:${project['plugins-gorm.hibernate5.version']}"
```

Comment:
This is org.grails.plugins:hibernate5

Reply:
```
if (artifactId.startsWith("gorm.")) {
    artifactId = artifactId.replace("gorm.", "")
}
```

was adjusting it before.

Reply:
fixed via changing the name in `gradle.properties`